packages/algob/src/runtime.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  addressToPk, mkTransaction,
+  parseSSCAppArgs, stringToBytes,
+  types,
+  uint64ToBigEndian
+} from "@algo-builder/runtime";
+
+export {
+  addressToPk,
+  mkTransaction,
+  parseSSCAppArgs,
+  stringToBytes,
+  uint64ToBigEndian,
+  types
```

Comment:
I wasn't able to export some specific types (eg. `ExecParams`..), using
```
export {
 ..
 types: { ExecParams, .. }
 }
 ```
because then we're using type as a value (gives type assertion error). 
 
Types declared in runtime api docs, are clickable in algob api docs only when i exported all types (like done above). Looks like this in algob api docs:
![image](https://user-images.githubusercontent.com/33264364/118887647-41d89a00-b918-11eb-9714-f841bd86c7ad.png)